### PR TITLE
Fixed Grammatical Errors

### PR DESCRIPTION
**Pull request by JesseSumrak**

### CODE REVIEW

**Comment by jmayntzhusen** [comment]
I am not strong enough in the English grammar rules for this, but it seems your edit has way too many commas in that first sentence, perhaps @marcemarc can weigh in? 😄

**Comment by shinypenny31** [comment]
Hahaha I feel you @jmayntzhusen! But as a "list," all items within a list need to be separated by a comma. The last comma in the series is optional as the Oxford comma. Most docs prefer the Oxford comma because it improves the reading experience (Oxford commas were originally omitted for newspapers to fit more content on a page by removing commas), but if your docs don't do Oxford comma I can remove.

File: Reference/Templating/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ By default Umbraco uses MVC templates.
 
 #### [Working with MVC (views, razor, etc...)](Mvc/index.md)
 
-Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.
+Describes how to work with MVC views, the razor syntax, and APIs available, as well details on how to create forms, step-by-step guides and other advanced techniques.
```
Comment (marcemarc) [inline Reference/Templating/index.md]: While we're adding some commas in here, how about we tweak the text too?

'as well details on how to create forms' 

could become

'as well as details on how to create forms'

etc

File: Reference/Templating/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ If you prefer to work with Webforms, you can change the default template engine
 
 #### Hybrid templates
 
-Umbraco can work with both MVC and Webforms templates at the same time, however there can only be one default. Umbraco will first check for an MVC view file before it checks for a Webforms master page file. For example, if a template was created in the backoffice called "Home", then when rendering a content page with this template Umbraco will check these locations and use the first one that is found:
+Umbraco can work with both MVC and Webforms templates at the same time; however, there can only be one default. Umbraco will first check for an MVC view file before it checks for a Webforms master page file. For example, if a template was created in the backoffice called "Home", then when rendering a content page with this template Umbraco will check these locations and use the first one that is found:
```
Comment (marcemarc) [inline Reference/Templating/index.md]: would this read better?
Umbraco will check for an MVC view before it checks for a Webforms Master Page. For example, if a template was created in the backoffice called 'Home', then when rendering a content page with this template, Umbraco will check the following locations in this order and use the one that is found first:

**Comment by nul800sebastiaan** [comment]
Hi there @shinypenny31,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Comment by jmayntzhusen** [comment]
Hi @JesseSumrak 

As we have seen no responses to our suggestions we will close this PR, if you get some time to fix it in the future you can reopen it 🙂

**Pull request closed without merging**